### ISSUE TITLE
Register Chronoline with package manages

### ISSUE THREAD

**Issue by Padhraic** [StoicLoofah/chronoline.js#33]
I think we could register Chronoline as package on bower, npm and nuget.
It would help users get and update the package more easily. Also we could control what version is the current stable release. At the moment I think most people are going to the github master branch witch would usually be pre-release or rc.

**Comment by StoicLoofah** [comment]
sure. we can definitely tackle this after #32 gets us a more up-to-date tagged release in 0.1.2

**Comment by Padhraic** [comment]
#32 will have minification so we can wait for that.

**Comment by Padhraic** [comment]
I have registered chronoline the the three package mangers below.

bower: http://bower.io/search/?q=chronoline
npm: https://www.npmjs.com/package/chronoline
Nuget: https://www.nuget.org/packages/chronoline/

Couple of points,
@StoicLoofah register with npm and nuget and I can set you as collaborator/owner.
There is a discrepancy between the versions, bower is latest is 0.1.2 and npm and nuget are 0.1.3. That is my bad, npm took the master package.json ver num when I published. When we tag v0.1.3 bower will update to match.
I published npm before creating the .npmignore so test and other folders are included in current npm package. The ignore will be applied on the next version.
